nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ValidateRecord.java

```diff
@@ -207,6 +225,8 @@
         properties.add(SCHEMA_TEXT);
         properties.add(ALLOW_EXTRA_FIELDS);
         properties.add(STRICT_TYPE_CHECKING);
+        properties.add(MAX_VALIDATION_DETAILS_LENGTH);
```

Comment:
Although the order of properties is not guaranteed (in the UI, e.g.), they tend to show up in the order they were added. Though not a requirement, I recommend switching the order of these properties just for clarity, totally up to you though.

Reply:
Done